### PR TITLE
Small email enhancements

### PR DESCRIPTION
**Pull request by vvdimako**
Hi there.
I have made a couple of email enhancements on TaskBoard, maybe you consider them useful: a) improved the email templates a bit and added a new function to correctly produce the board URL and b) removed the actors from the mailing lists (e.g. the user that adds/edits an item does not get a notification email).

### CODE REVIEW

File: api/mail_templates/editComment.html [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+<b>Item</b>: %title%<br><br>
+
+<u><b>Comment</b></u>:<br>
+<i>
+%comment%
+</i><br>
 <br>
-<a href="http://%hostname%/#/boards/%boardid%">Navigate to board!</a>
+<br>
+-- from the PPG TaskBoard
```
Comment (kiswa) [inline api/mail_templates/editComment.html]: This branding should not be in default TaskBoard.

File: api/mail_templates/newComment.html [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
+<b>Item</b>: %title%<br><br>
+
+<u><b>Comment</b></u>:<br>
+<i>
+%comment%
+</i><br>
 <br>
-<a href="http://%hostname%/#/boards/%boardid%">Navigate to board!</a>
+<br>
+-- from the PPG TaskBoard
```
Comment (kiswa) [inline api/mail_templates/newComment.html]: Same branding issue.

**Comment by kiswa** [comment]
This is helpful, thank you.

However, PRs should be made against the `dev` branch and not `master`.

I have clarified this in the future `re-write` branch.

**Comment by vvdimako** [comment]
Sorry about that, didn't notice! I'll checkout the dev branch.

**Comment by kiswa** [comment]
No worries, that's why I added things to the `re-write` branch to make it clearer in the future. Many contributors didn't notice.

**Pull request closed without merging**